PoSSapi/src/Application/Businesses/Commands/CreateBusinessCommand.cs

```diff
@@ -0,0 +1,49 @@
+using AutoMapper;
+using MediatR;
+using PoSSapi.Application.Common.Exceptions;
+using PoSSapi.Application.Common.Interfaces;
+using PoSSapi.Domain.Entities;
+
+namespace PoSSapi.Application.Businesses.Commands;
+
+public class CreateBusinessCommand : IRequest<Guid>
+{
+    public Guid Id { get; init; }
+    public Guid? ManagerId { get; init; }
+}
```

Comment:
Perhaps businessLocation should also be here and in the update command? Also, it would be nice for the business to have a name, but that was also missing from the swagger, so perhaps it's not a necessity.